docs/content/guides/getting-started/binding-to-data.md

```diff
@@ -219,40 +209,55 @@ const hot = new Handsontable(container, {
 ::: only-for react
 ::: example #example3 :react
 ```jsx
-import React, { Fragment, useEffect } from 'react';
+import React from 'react';
 import ReactDOM from 'react-dom';
 import { HotTable } from '@handsontable/react';
 import { registerAllModules } from 'handsontable/registry';
 
 // register Handsontable's modules
 registerAllModules();
 
-const ExampleComponent = () => {
-  const data = [
-    { id: 1, name: 'Ted Right', address: '' },
-    { id: 2, name: 'Frank Honest', address: '' },
-    { id: 3, name: 'Joan Well', address: '' },
-    { id: 4, name: 'Gail Polite', address: '' },
-    { id: 5, name: 'Michael Fair', address: '' },
-  ];
-  const hotSettings = {
-    data,
-    colHeaders: true,
-    height: 'auto',
-    width: 'auto',
-    minSpareRows: 1,
-    licenseKey: 'non-commercial-and-evaluation'
-  };
+const App = () => {
 
   return (
-    <Fragment>
-      <HotTable settings={hotSettings}>
-      </HotTable>
-    </Fragment>
+    <HotTable
+      data={[
```

Comment:
I don't know if moving larger datasets like this into a prop is a good idea - for me, personally, it's less readable than storing it in a variable and binding it to the prop. WDYT?

Reply:
For sure i'll be fixing formatting here to similar like below. Other than that its just a preference, w/e for me.

Reply:
I agree with @jansiegel. My reasoning is that `data={data}` is more convenient, because:
1. the reader can use all other props as they are (by copying from the snippet), except for `data`, which for sure will be different in their app
2. `data` is the most likely to come from another level in the application and passed here using a variable

Reply:
So whats the conclusion here?

Reply:
fixed in 53c8469ddc4f57cdeb17853c255456a7d0879508

Reply:
In that case we should change it in the entirety of the docs, right? There are a few more places in other sections where array is passed directly (like column summary, row sorting).

Reply:
Yes, I will add this to the style guide and make an issue about replacing that on other pages. We can do that in a separate PR with less priority